unix_integration/src/client.rs

```diff
@@ -37,7 +37,7 @@ impl Encoder<ClientRequest> for ClientCodec {
             error!("socket encoding error -> {:?}", e);
             IoError::new(ErrorKind::Other, "JSON encode error")
         })?;
-        debug!("Attempting to send request -> {:?} ...", data);
+        debug!("Attempting to send request -> {:?} ...", msg);
```

Comment:
Can we check that a password auth request with debug doesn't plaintext pw's in logs?